README.md

```diff
@@ -84,6 +85,12 @@ Default `.`
 
 This is the base directory for all of your SASS/SCSS files. Grunt will recurse this directory looking for color variables.
 
+#### excludedFiles
+Type: `String`;
+Default `/`
```

Comment:
This default seems a bit ambiguous to me. Could be interpreted as "exclude everything under the root". Maybe we could just say "Default null" in the docs (regardless of what the actual underlying default is).